shared/params/config.go

```diff
@@ -134,8 +134,8 @@ var defaultBeaconConfig = &BeaconChainConfig{
 
 	// Time parameter constants.
 	MinAttestationInclusionDelay:     1,
-	SecondsPerSlot:                   6,
-	SlotsPerEpoch:                    64,
+	SecondsPerSlot:                   12,
```

Comment:
Why?

Reply:
Updated descriptions, the rationale is more events will occur per slot in these modified proposals so upped 6 to 12